entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+# add node_modules/.bin to PATH
+export PATH=/app/node_modules/.bin:$PATH
```

Comment:
added reason: `/app/entrypoint.sh: line 3: i18nexus: not found` when docker build